packages/ui/src/molecules/RadioGroup/RadioGroup.test.tsx

```diff
@@ -0,0 +1,105 @@
+import { render } from '@testing-library/react'
+import { renderHook } from '@testing-library/react-hooks'
+import userEvent from '@testing-library/user-event'
+import type { ReactNode } from 'react'
+import React from 'react'
+
+import { RadioGroup, RadioOption, useRadioGroup } from '.'
+
+describe('RadioGroup', () => {
+  it('Should render radio group with radio option', () => {
+    const onChange = jest.fn()
+
```

Comment:
Can remove it and pass an inline noop function `() => {}`

Reply:
The onChange function shouldn't be required, IMO.

Reply:
will pass an inline function () => {} because the prop onChange is passed for RadioOptions, without it the tests emit an alert about his lack